### PR TITLE
fix(security): sanitize request parameters 2.8.x

### PR DESCRIPTION
**Pull request by jeremyjaouen**
## Description
this PR intends to sanitize request parameters.

**Fixes** # MON-7105

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [x] 2.8.x

## Checklist

- [x] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by adr-mo — APPROVED** [review]

**Review by kduret — APPROVED** [review]

File: www/include/configuration/configCentreonBroker/blockConfig.php [hunk 1]
```diff
@@ -45,9 +45,14 @@
  * Cast the block id in int
  */
 try {
-    $id = (string)$_GET['blockId'];
-    $tag = (string)$_GET['tag'];
-    $pos = (int)$_GET['pos'];
+    $id = filter_var((string)$_GET['blockId'], FILTER_SANITIZE_STRING);
```
Comment (callapa) [inline www/include/configuration/configCentreonBroker/blockConfig.php:48]: If the blockId parameter does not exist, a notice message (Undefined index) will appear in the PHP log file.

File: www/include/configuration/configCentreonBroker/blockConfig.php [hunk 2]
```diff
@@ -45,9 +45,14 @@
...
+    $tag = filter_var((string)$_GET['tag'], FILTER_SANITIZE_STRING);
```
Comment (callapa) [inline www/include/configuration/configCentreonBroker/blockConfig.php:49]: Same thing here

File: www/include/configuration/configCentreonBroker/blockConfig.php [hunk 3]
```diff
@@ -45,9 +45,14 @@
...
+    $pos = filter_var((int)$_GET['pos'], FILTER_VALIDATE_INT);
```
Comment (callapa) [inline www/include/configuration/configCentreonBroker/blockConfig.php:50]: Same thing here

**Review by callapa — CHANGES_REQUESTED** [review]

File: www/include/configuration/configCentreonBroker/blockConfig.php (hunk 1 above)
Comment (jeremyjaouen) [reply www/include/configuration/configCentreonBroker/blockConfig.php:48]: those checks are handled l.40

```
if (!isset($_GET['tag']) || !isset($_GET['pos']) || !isset($_GET['blockId'])) {
    exit();
}
```

**Review by callapa — APPROVED** [review]

**Pull request merged**